recipes/TIES20/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "TIES20" %}
+{% set version = "20.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/UCL-CCS/TIES20/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4af8f09bc480467ef8941dbc4f1a96edadb1dad6111929fc1b664eb67d266ef7
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+  entry_points:
+    - ties = ties:cli.command_line_script
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python >=3.7
+    - pip
+    - numpy
+    - cython
+  run:
+    - python >=3.7
+    - ambertools >=20.0,<=22.0
+    - libgfortran
+    - ties_md
+    - numpy
+    - cython
+    - setuptools
+    - matplotlib-base
+    - networkx
+    - dimod
+    - tabulate
+
+test:
+  imports:
+    - ties
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
+    - python -c "import pytest; pytest.main(['./TIES20-{{ version }}/testing'])"
```

Comment:
Where is `./TIES20-{{ version }}/testing` located? I don't see it in the recipe directory.

Reply:
It's in the repository under `testing`. Can we use the env variable `SRC_DIR` to access it? Ie 
```
- pytest $SRC_DIR/testing
```

Reply:
If you want these files available for testing, then you should add them to `test/source_files`

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#source-files